### DISCUSSION TITLE
怎样显示成果列表页？

### DISCUSSION THREAD

**Discussion by CandyTt20** [nju-lug/NJUThesis#108] [Q&A] [answered]
用户手册里只有个`\njupaperlist[⟨标题⟩]{⟨文献⟩}`命令，具体如何使用？

**Comment by atxy-blip** [comment, marked as answer]
可以参考 [test/chapters/Achievements.tex](https://github.com/nju-lug/NJUThesis/blob/master/test/chapters/Achievements.tex) 这个文件的写法。

`\njupaperlist[⟨标题⟩]{⟨文献⟩}` 会生成一个不编号的 section，第一个可选参数是 section 标题，第二个参数的用法同 `\cite{⟨文献⟩}`，为 `.bib` 文件中的标签。
